js/src/tooltip.js

```diff
@@ -13,11 +13,11 @@ import Util from './util'
 const Tooltip = (($) => {
 
   /**
-   * Check for Tether dependency
-   * Tether - http://tether.io/
+   * Check for Popper dependency
+   * Popper - https://popper.js.org
    */
-  if (typeof Tether === 'undefined') {
-    throw new Error('Bootstrap tooltips require Tether (http://tether.io/)')
+  if (typeof Popper === 'undefined') {
+    throw new Error('Bootstrap tooltips require Popper (https://popper.js.org)')
```

Comment:
Sure I'll change that 😉 
I have to update those lines : https://github.com/twbs/bootstrap/blob/v4-dev/grunt/npm-shrinkwrap.json#L3795-L3798

Do you know the correct url for `resolved` ?